GitNotes.txt

```diff
@@ -0,0 +1,9 @@
+Create Repo --->
```

Comment:
This commit just contains this text file? The rest of your proj?

Reply:
I think you've opened a PR on top of your initial commit. What you can do is to open up a new Repo then commit your original work to that new repo and open a PR and I can review that? Otherwise I can't review work that's already in the master branch if that makes sense. 

1 - open a new repo
2 - commit something small and make that your first commit. Maybe the text file here. 
3 - Commit your original work on that and open a PR and then I can review all of that.

How's that sound?